docs/overview/proof-of-burn.md

```diff
@@ -0,0 +1,36 @@
+# Proof-of-Burn
+
+Proof-of-Burn (PoB) is the consensus algorithm used by the Koinos blockchain. It shares properties with both Proof-of-Work (PoW) and Proof-of-Stake (PoS). At a high level, Proof-of-Burn emulates the behavior of Proof-of-Work while keeping the concept of real world assets such as mining rigs virtualized on-chain.
```

Comment:
"while" implies a contrast. I would say "by keeping the concept of real world assets with virtualized on-chain mining rigs" or something similar. Currently the second part of the sentence feels like it should contradict the emulation of PoW, when it is justifying the comparison.